### DISCUSSION TITLE
looking for an example of using TimeseriesForecaster

### DISCUSSION THREAD

**Discussion by ilanaklo** [keras-team/autokeras#1499] [Q&A]
I'm trying to build a network for time series forecasting and I came across this TimeseriesForecaster class, but I didn't find any usage example for it.
Can anybody give me a working example for it?
Thanks!

**Comment by abgese** [comment]
Hi I'm currently working on a tutorial for this class. For now I recommend looking at the unit tests to get a general idea of how to use the API.

**Reply by Gorbov** [reply]
Hi!
Are there any news about the tutorial?
